### ISSUE TITLE
Game Crash

### ISSUE THREAD

**Issue by AmibeSkyfy16** [Pedro270707/headed#4]
The game will crash after tried to type << bee >> in the creative menu items

Logs: 
[link1](https://pastebin.com/EhVwnrNW)
[l2](https://pastebin.com/WuBpaMwB)
[image](https://pasteboard.co/zibeR3Ca9R8F.png)

Version: Latest (latest fabric 1.19.4)

**Comment by J-F2** [comment]
are you using the latest version of the mod? this happened to me when i was using mismatched mod and fabric versions.

**Comment by Pedro270707** [comment]
I couldn't replicate the issue. Is it consistent or did it only happen once?

**Comment by AmibeSkyfy16** [comment]
Hmm that's strange, I tried again and no more bug, but I also had a doubt about the "Variety Aquatic" mod and putting it on too, I have the error again. I will make an issue for "Variety Aquatic" then. Except that I also tried again by deactivating "Variety Aquatic" and there, the bug reappears. 
Found it, it seems that it comes from GeckoLib, the mod was automatically downloaded because it is a dependency of Variety Aquatic and I had not seen, but by disabling it too, the bug disappears
